darkdetect/_linux_detect.py

```diff
@@ -4,25 +4,20 @@
 #  Distributed under the terms of the 3-clause BSD License.
 #-----------------------------------------------------------------------------
 
-from ctypes import util, cdll, c_char_p, byref
+import subprocess
 
 
 def theme():
     # Here we just triage to GTK settings for now
-    try:
-        gtklib = cdll.LoadLibrary(util.find_library('gtk-3'))
-        gtklib.gtk_init(None, None)
-        settings = gtklib.gtk_settings_get_default()
-        res = c_char_p()
-        gtklib.g_object_get(settings, b"gtk-theme-name", byref(res), 0)
-        theme = res.value.decode()
-    except Exception:
-        return 'Light'
+    out = subprocess.Popen(
```

Comment:
Can we use `subprocess.run()` here? `run()` is "The recommended approach to invoking subprocesses [...] for all use cases it can handle", and I do not see anything special in this call.